src/applications/medical-copays/components/Alerts.jsx

```diff
@@ -8,7 +8,11 @@ import Telephone, {
 const Alert = ({ children }) => children;
 
 Alert.Error = () => (
-  <va-alert class="row vads-u-margin-bottom--5" status="error">
+  <va-alert
+    class="row vads-u-margin-bottom--5"
+    status="error"
+    data-e2e-id="error-alert"
```

Comment:
hey @tlei123 👋  .. noticed were using `data-e2e-id` here but the [docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/Cypress-Best-Practices-on-VA.gov-Platform.1870430211.html#CypressBestPracticesonVA.govPlatform-data-testidAttribute) recommend using `data-testid` so wanna confirm which test id we should be using going forward?

Reply:
good catch, @craigwheeler !  i'll update.  didn't know Platform's decided on a standard.